# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/blindcoder/libsdl/alsa-fix.patch
# ROCK Linux is Copyright (C) 1998 - 2004 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---


Fix outdated ALSA API useage ...

  - Rene Rebe <rene@exactcode.de>

--- SDL-1.2.6/src/audio/alsa/SDL_alsa_audio.c.orig	2004-01-11 20:16:20.000000000 +0100
+++ SDL-1.2.6/src/audio/alsa/SDL_alsa_audio.c	2004-01-11 20:39:32.000000000 +0100
@@ -34,6 +34,11 @@
 #include <sys/types.h>
 #include <sys/time.h>
 
+/* until convertion to the new ALSA API ... */
+
+#define ALSA_PCM_OLD_HW_PARAMS_API
+#define ALSA_PCM_OLD_SW_PARAMS_API
+
 #include "SDL_audio.h"
 #include "SDL_error.h"
 #include "SDL_audiomem.h"
